src/Sarif/ExtensionMethods.cs

```diff
@@ -288,7 +288,7 @@ public static string GetMessageText(this Result result, ReportingDescriptor rule
             return GetMessageText(result, rule, concise: false);
         }
 
-        public static string GetMessageText(this Result result, ReportingDescriptor rule, bool concise = false)
+        public static string GetMessageText(this Result result, ReportingDescriptor rule, bool concise = false, int maxLength = 200)
```

Comment:
>200 [](start = 128, length = 3)

is 120 or 128 a better default?

Reply:
When I filed the issue I said 120; when I coded it, I worried it was too short. But OK, anybody who calls the API and decides it's not enough can always increase it.

---
In reply to: [445546182](https://github.com/microsoft/sarif-sdk/pull/1932#discussion_r445546182) [](ancestors = 445546182)

Reply:
The key question is 'too short for what'? i.e., under what circumstances would someone be likely to request a concise string and expect something around 200 characters. If you have a scenario in mind, let me know. I've mostly been thinking about flat list views in IDEs as a driver for this API, or work item titles. In these contexts, 200 chars seems more than what's required. Open to thinking about other UX scenarios if you have one.

---
In reply to: [445558418](https://github.com/microsoft/sarif-sdk/pull/1932#discussion_r445558418) [](ancestors = 445558418,445546182)